bitnami/wavefront/templates/collector-daemonset.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if and .Values.collector.enabled .Values.collector.useDaemonset }}
+apiVersion: apps/v1
```

Comment:
In general I miss several of our standardized values. Just to name a few:

   - containerSecurityContext
   - podSecurityContext
   - collector.livenessProbe
   - collector.readinessProbe
   - collector.customLivenessProbe
   - collector.customReadinessProbe
   - collector.podLabels
   - collector.podAnnotations
   - collector.sidecars
   - collector.initContainers
   - collector.extraEnvVars
   - collector.extraEnvVarsCM
   - collector.extraEnvVarsSecret
   - collector.command (which would override the default one)
   - collector.args (which would override the default one)
   - collector.extraVolumes
   - collector.extraVolumeMounts
   - collector.tolerations (that would override the default one)
   - collector.affinity
   - collector.lifecycleHooks
   (the same with proxy)
   - commonAnnotations
   - commonLabels
   - extraDeploy

   I'm missing others, I think

Reply:
I recommend you to check harbor or the recent webapps. You will see the snippets to copy-paste.